lld/test/ELF/riscv-section-layout.s

```diff
@@ -33,7 +34,9 @@
 # NOSDATA-DAG:  [[#]]: {{0*}}[[#BSS]]         0 NOTYPE  GLOBAL DEFAULT [[#]] (.bss) __bss_start
 # NOSDATA-DAG:  [[#]]: {{0*}}800              0 NOTYPE  GLOBAL DEFAULT  1 (.dynsym) __global_pointer$
 
-# CHECK:      .text
+# CHECK:      .rodata
+# CHECK-NEXT: .srodata
```

Comment:
We might need  `.srodata{{$}}` as `.srodata.suffix` would match as well.

Reply:
Good point. I'll do this for all the sections that should end up combined.

Reply:
`{{$}}` didn't work, so I just chose to end all the checked sections with their PROGBITS/NOBITS/DYNAMIC type.